core/templates/components/rubrics-editor/rubrics-editor.component.ts

```diff
@@ -95,6 +95,39 @@ export class RubricsEditorComponent {
     return Boolean(this.editableExplanations[difficulty][index]);
   }
```

Comment:
Maybe worth renaming `isExplanationValid` to `isExplanationEmpty`, since that really is what it is checking. I know you didn't modify this function, but they can now use extra clarity.

Reply:
There is already a function of name `isExplanationEmpty`, we have to choose some other name.
Thanks

Reply:
Oh I didn't see that. In that case, we should remove `isExplanationValid` from the code -- it is like a duplicate, though the code is not exactly the same. We can just change the existing `isExplanationEmpty` to have this signature:
```
isExplanationEmpty(difficulty: string, index: number): boolean
```

Reply:
Sure,
Thanks

Reply:
So, if we change the `isExplanationEmpty` then it will not work for `<p><p>` right? because currently `isExplanationEmpty` is as follows
```
isExplanationEmpty(explanation: string): boolean {
    return explanation === '<p></p>' || explanation === '';
  }
```

Reply:
Let's keep `isExplanationEmpty` as is -- so don't change the code you pasted above. Just use `!isExplanationEmpty` instead of `isExplanationValid`. Does that make sense? Let me know if I'm missing something.

Reply:
Our new definition of "empty" should be `explanation === '<p></p>' || explanation === ''`

Reply:
Done